### PR TITLE
Remove non-public iris endpoints

### PR DESCRIPTION
**Pull request by hyehan**
Remove non-public iris endpoints from the chain-registry.

### CODE REVIEW

File: irisnet/chain.json [hunk 1]
```diff
@@ -8,9 +8,7 @@
   "bech32_prefix": "iaa",
   "daemon_name": "iris",
   "node_home": "$HOME/.iris",
-  "key_algos": [
-    "secp256k1"
-  ],
+  "key_algos": ["secp256k1"],
```
Comment (reversesigh) [inline irisnet/chain.json:11]: Can you edit to maintain the original formatting on this and compatible_versions please?

Comment (hyehan) [reply irisnet/chain.json:11]: Changed back to the original format, thanks @reversesigh

Comment (JeremyParish69) [reply irisnet/chain.json:11]: why is this duplicating the key_algos property?

**Review by dylanschultzie — CHANGES_REQUESTED** [review]
@hyehan please remove duplicate `key_algos` property

**Pull request merged**